### PR TITLE
add overwolf

### PR DESCRIPTION
**Pull request by solomoncyj**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/74997)

### CODE REVIEW

**Comment by solomoncyj** [comment]
@denelon zips

**Comment by russellbanks** [comment]
The manifests for Zip files in the future will be more nuanced than this, I believe, such as specifying what the nested installer is, the content setup inside the zip file, etc. I suggest it's best to wait until Winget 1.4 is out and the associated tools have been updated to support Zip files.

**Review by ItzLevvie — DISMISSED** [review]

**Review by ItzLevvie — APPROVED** [review]

File: manifests/o/Overwolf/Overwolf/0.204.0.1/Overwolf.Overwolf.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Created with YamlCreate.ps1 v2.1.4 $debug=NVS2.7-3-0-preview-7
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Overwolf.Overwolf
+PackageVersion: 0.204.0.1
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
```
Comment (ItzLevvie) [inline manifests/o/Overwolf/Overwolf/0.204.0.1/Overwolf.Overwolf.installer.yaml:7]:
```suggestion
InstallerType: zip
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/o/Overwolf/Overwolf/0.204.0.1/Overwolf.Overwolf.installer.yaml (hunk 1 above)
Comment (solomoncyj) [reply manifests/o/Overwolf/Overwolf/0.204.0.1/Overwolf.Overwolf.installer.yaml:7]: but

**Review by ItzLevvie — APPROVED** [review]

**Pull request closed without merging**